CMakeLists.txt

```diff
@@ -1,11 +1,7 @@
 # CMake build script for ZeroMQ
 project(ZeroMQ)
 
-if(${CMAKE_SYSTEM_NAME} STREQUAL Darwin)
-  cmake_minimum_required(VERSION 3.0.2)
-else()
-  cmake_minimum_required(VERSION 2.8.12)
-endif()
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
Why are you dropping compatibility with 2.8? We definitely do not want to do that

Reply:
```
CMake Deprecation Warning at tests/CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
```

Reply:
And? Just ignore it

Reply:
IMHO: **Warnings are Errors**!

On every system customer may install an actual cmake version with:

`pip install cmake`

Reply:
No, these are not compiler errors, we do not break backward compatibility